plotly/offline/offline.py

```diff
@@ -129,6 +129,8 @@ def init_notebook_mode(connected=False):
         script_inject = (
             ''
             '<script>'
+            'window.is_loaded = false;'
+            'setTimeout(function(){{window.is_loaded=true}},100);'
```

Comment:
Yeah I could change that if we decide to stick with it. Another possible option could be to encode the time in the script. This way, we could put a condition such that the script will not run if some amount of time `c` has passed. Therefore on page refreshes the embedded js shouldn't run until the next time the cell is ran.